packages/manager/apps/web/client/app/domain/webhosting/order/steps/payment/domain-webhosting-order-steps-payment.html

```diff
@@ -2,7 +2,7 @@
     data-header="{{:: 'domain_webhosting_order_payment_step_title' | translate }}"
     data-on-focus="$ctrl.getCheckout()"
     data-loading="$ctrl.stepper.loadingCheckout"
-    data-valid="$ctrl.defaultPaymentMethod != null && $ctrl.agreeContracts"
+    data-valid="$ctrl.agreeContracts"
```

Comment:
**question:** Do we have validation from PO to remove the check if default payment method does not exist?

Reply:
I didn't ask but the current behavior blocks users if they want to pay with another payment method or if they have not.
AS UI allows both choices, the button has to do the same.